drivers/ng_at86rf2xx/ng_at86rf2xx_getset.c

```diff
@@ -139,13 +139,16 @@ void ng_at86rf2xx_set_freq(ng_at86rf2xx_t *dev, ng_at86rf2xx_freq_t freq)
     uint8_t tmp1 = 0, tmp2 = 0;
     tmp1 = ng_at86rf2xx_reg_read(dev, NG_AT86RF2XX_REG__TRX_CTRL_2);
     tmp1 &= ~(NG_AT86RF2XX_TRX_CTRL_2_MASK__FREQ_MODE);
+    tmp2 = ng_at86rf2xx_reg_read(dev, NG_AT86RF2XX_REG__RF_CTRL_0);
+    /* Erase previous conf for GC_TX_OFFS */
+    tmp2 &= ~0x03;
```

Comment:
Just seeing this now, aren't there mask defines you could use here instead of magic numbers? That would greatly improve consistency and readability.  Same for l151 and l162.